tensorflow/cc/framework/gradients.cc

```diff
@@ -167,11 +167,12 @@ Status SymbolicGradientBuilder::BackpropAlongEdge(const Output& dst_grad,
 std::vector<bool> SymbolicGradientBuilder::GetReachableNodes() {
   std::vector<bool> reachable_nodes(scope_.graph()->num_node_ids(), false);
   std::deque<Node*> queue;
-  std::vector<bool> visited(scope_.graph()->num_node_ids(), false);
   for (const Output& out : outputs_) {
-    if (!reachable_nodes[out.node()->id()]) {
-      queue.push_back(out.node());
-      reachable_nodes[out.node()->id()] = true;
+    Node* const out_node = out.node();
+    const int out_node_id = out_node->id();
```

Comment:
nit: I think creating these out variables doesn't increase readability and makes the code longer. Can you change it back to using the inline function calls?

Reply:
done